build.sbt

```diff
@@ -187,7 +187,7 @@ lazy val example = (project in file("example"))
     commands += runStreamExampleWithOtelAgent,
     commands += runZioExampleWithOtelAgent
   )
-  .dependsOn(extension)
+  .dependsOn(core, extension)
```

Comment:
Extension is still needed so that we see the akka cluster metrics. I will work on a separate PR to decouple this - I didn't want to do too much now